docs/appendices/release-notes/5.4.4.rst

```diff
@@ -66,3 +66,6 @@ Fixes
 
 - Fixed an issue with missing validation on ``INSERT`` statement, allowing to
   specify duplicate target columns.
+
+- Fixed an issue that caused ``UPDATE`` statements, invoked immediately after a
+  ``DELETE`` statement, which empties a table, to show an error.
```

Comment:
Isn't the thing about running a delete kinda misleading? It's about an empty table, independent of whether it was just created and therefore empty, or empty because of deletes. No?